ui/src/App.tsx

```diff
@@ -169,15 +169,32 @@ const App = () => {
 
   const openModal = (arg: EventClickArg) => {
     const entryId = arg.event._def.extendedProps._id;
-    getEntryDetails(entryId);
+    const start = arg.event._def.extendedProps.entryStart;
+    getEntryDetails(entryId, start);
+  };
+
+  const setEventsWithExtraFields = (entries: any) => {
```

Comment:
Are there other extra fields we plan to add in the future? If not, can we just call this something more simple like `setEventsWithStart`. Also, can we not use `event` in the name in favor of `entry` for clarity?

Reply:
Sure can update